### PR TITLE
add mmark formula

### PR DESCRIPTION
**Pull request by dericed**

### CODE REVIEW

File: Formula/mmark.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    ln_sf buildpath, buildpath/"src/github.com/miekg/mmark"
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    cd "mmark" do
+      system "go", "build", "-o", bin/"mmark"
+    end
+  end
+
+  test do
+    system "#{bin}/mmark", "-version"
```
Comment (bfontaine) [inline Formula/mmark.rb]: Please write a test that does something more substantial than printing the version. You can create a small markdown file and process it with `mmark`.

File: Formula/mmark.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+class Mmark < Formula
+  desc "Powerful markdown processor in Go geared towards the IETF"
+  homepage "https://github.com/miekg/mmark"
+  url "https://github.com/miekg/mmark/archive/v1.3.4.tar.gz"
+  sha256 "e03744da8d16cc742423685e2ad7cb1af61bf6dc5364c6875057b7c28ab26bb8"
+
+  depends_on "go" => :build
+
+  go_resource "github.com/BurntSushi/toml" do
+    url "https://github.com/BurntSushi/toml.git", :revision => "443a628bc233f634a75bcbdd71fe5350789f1afa"
```
Comment (bfontaine) [inline Formula/mmark.rb]: Please move the `:revision` on the next line to keep the text at a reasonable width:

```rb
url "https://github.com/BurntSushi/toml.git",
  :revision => "443a628bc233f634a75bcbdd71fe5350789f1afa"
```

**Review by bfontaine — CHANGES_REQUESTED** [review]

File: Formula/mmark.rb (hunk 2 above)
Comment (zmwangx) [inline Formula/mmark.rb]: Actually all `go_resource` revisions are aligned to the opening quote at the moment.

**Comment by woodruffw** [comment]
Closing due to inactivity.

**Pull request closed without merging**